src/server/node_manager.c

```diff
@@ -724,6 +718,12 @@ node_down_requeue(struct work_task *pwt)
 						/* notify all sisters to discard the job */
 						discard_job(pj, "on node down requeue", 0);
 
+						/* Clear "resources_used" before job is requeued */
```

Comment:
Comment needs correction since we are clearing the resources after the job is requeued.